### PR TITLE
Feat/UI bugs light theme

### PR DESCRIPTION
**Pull request by NikIvv**

### CODE REVIEW

File: src/components/DelegatesTable/DelegateVoteCheckbox.vue [hunk 1]
```diff
@@ -93,13 +93,14 @@ export default {
...
-.dark {
-  .delegate-vote-checkbox {
-    &__icon {
+      &--good {
+        color: map-get($adm-colors, 'good');
+      }
+
+      &--danger {
+        color: map-get($adm-colors, 'danger');
+      }
```
Comment (bludnic) [inline src/components/DelegatesTable/DelegateVoteCheckbox.vue:103]: Thats correct, theme specific styles like `color` and `background-color` should be inside `.v-theme--light` and `.v-theme--dark` selectors.

1. So, you can remove `&--good` and `&--danger` classes from common styles (line 82-88)
2. Check if default colors in dark theme are ok. If not, define them in `.v-theme--dark`

File: src/components/LoginForm.vue [hunk 2]
```diff
@@ -72,6 +72,7 @@ export default defineComponent({
     const router = useRouter()
     const store = useStore()
     const { t } = useI18n()
+    const className = 'login-form'
```
Comment (bludnic) [inline src/components/LoginForm.vue:75]: Define `classes` outside Vue component. See example: https://github.com/Adamant-im/adamant-im/blob/67cb2a6e881f82e6330e4af321784f292a966b8b/src/components/nodes/NodesTable.vue#L96-L101

**Review by bludnic — CHANGES_REQUESTED** [review]

File: src/components/DelegatesTable/DelegateVoteCheckbox.vue (hunk 1 above)
Comment (NikIvv) [reply src/components/DelegatesTable/DelegateVoteCheckbox.vue]: Done.

File: src/components/LoginForm.vue (hunk 2 above)
Comment (NikIvv) [reply src/components/LoginForm.vue]: Done in [4dd0a94](https://github.com/Adamant-im/adamant-im/pull/679/commits/4dd0a94ce58e78200a7e9f5c4fad44bb2ea661f0)

**Review by bludnic — CHANGES_REQUESTED** [review]

**Comment by bludnic** [comment]
[UI bug: Light theme](https://trello.com/c/zGFNXZwR/563-ui-bug-light-theme)

**Comment by bludnic** [comment]
[UI bug: Light theme](https://trello.com/c/zGFNXZwR/563-ui-bug-light-theme)

File: src/components/LoginForm.vue [hunk 3]
```diff
@@ -73,6 +73,11 @@ export default defineComponent({
     const store = useStore()
     const { t } = useI18n()
     const showSpinner = ref(false)
+    const className = 'login-form'
+    const classes = {
+      root: className,
+      textField: `${className}__textfield`
+    }
```
[the remark this answers is not in the document]
Comment (NikIvv) [reply src/components/LoginForm.vue]: Done.

File: src/components/DelegatesTable/DelegateVoteCheckbox.vue [hunk 4]
```diff
@@ -78,28 +78,34 @@ export default {
...
-      color: map-get($adm-colors, 'danger');
-    }
   }
 }
 
+/** Themes **/
 .v-theme--light {
   .delegate-vote-checkbox {
     &__icon {
-      color: map-get($adm-colors, 'muted');
```
[the remark this answers is not in the document]
Comment (NikIvv) [reply src/components/DelegatesTable/DelegateVoteCheckbox.vue:95]: There are some questions to be discussed then as 'muted' has been defined for light theme only in $adm-colors.

**Review by bludnic — APPROVED** [review]

**Pull request merged**